examples/maps_heatmaps/src/sidepanel.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
```

Comment:
This is example code, so don't commit it please, or at least without the `ExampleCompnent`

Reply:
ok done, we removed it.